clusterloader2/testing/density/constraints_100.yaml

```diff
@@ -0,0 +1,24 @@
+coredns:
+  cpuConstraint: 0.01
+  memoryConstraint: 9437184 #9 * (1024 * 1024)
+fluentd-gcp:
+  cpuConstraint: 1
+  memoryConstraint: 220200960 #210 * (1024 * 1024)
+heapster:
+  cpuConstraint: 0.1
+  nemoryConstraint: 340787200 #325 * (1024 * 1024)
+kube-apiserver:
+  cpuConstraint: 1.7
+  memoryConstraint: 975175680 #930 * (1024 * 1024)
+kube-controller-manager:
+  cpuConstraint: 0.9
+  memoryConstraint: 241172480 #230 * (1024 * 1024)
+kube-proxy:
+  cpuConstraint: 0.01
+  memoryConstraint: 10485760 #10 * (1024 * 1024)
```

Comment:
Same here - 10MB seems too aggresive to me.

Reply:
Changed to 30.